src/service/url-replacements-impl.js

```diff
@@ -76,7 +88,7 @@ export class UrlReplacements {
     this.initialized_ = true;
     // Returns a random value for cache busters.
     this.set_('RANDOM', () => {
-      return Math.random();
+      return String(Math.random());
```

Comment:
I suppose we could also support arrays of these values and automatically join them but that seems excessive